packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/upgrade/upgrade.html

```diff
@@ -16,7 +16,7 @@
 <oui-stepper data-current-index="$ctrl.currentStep">
     <oui-step-form
         data-header="{{:: ('dedicated_server_dashboard_upgrade_select_option_title_' + $ctrl.upgradeType) | translate }}"
-        data-description="{{:: ('dedicated_server_dashboard_upgrade_select_option_description_' + $ctrl.upgradeType) | translate }}"
+        data-description="{{:: $ctrl.hasRenewPeriod ? 'dedicated_server_dashboard_upgrade_select_option_description_' + $ctrl.upgradeType + '_with_interval' : 'dedicated_server_dashboard_upgrade_select_option_description_' + $ctrl.upgradeType | translate : { interval: $ctrl.renewPeriod } }}"
```

Comment:
regarding `hasRenewPeriod` function, i think you have to use it like:
```suggestion
        data-description="{{:: $ctrl.hasRenewPeriod() ? 'dedicated_server_dashboard_upgrade_select_option_description_' + $ctrl.upgradeType + '_with_interval' : 'dedicated_server_dashboard_upgrade_select_option_description_' + $ctrl.upgradeType | translate : { interval: $ctrl.renewPeriod } }}"
```